src/sacred_scripts/spk_eval.py

```diff
@@ -12,28 +14,72 @@ def config():
     """
     Settings for the evaluation script.
     """
+    in_path = './data/md17/ethanol.db'      # path to input file
     out_path = './results.db'               # path to output file
-    model_path = './training/best_model'    # path to trained model
+    model_dir = './training'                # path to trained model
```

Comment:
This default also seems dangerous...

Reply:
I forgot to remove my test settings. Should be ok now.